core/src/main/kotlin/br/com/concrete/yosef/api/property/spacing/MarginPropertyCommand.kt

```diff
@@ -39,11 +40,20 @@ class MarginPropertyCommand : DynamicPropertyCommand {
             dynamicProperty.value
                 .split(",")
                 .map {
-                    it.trim().toInt().dp(view.context)
+                    convertValueToDp(it, view.context)
```

Comment:
Maybe you should check if there is 4 values here, if there's not throw an especific exception message

Reply:
done, nice thinking.